qa/acceptance-tests/src/test/java/io/camunda/it/migration/IdentityMigrationTestUtil.java

```diff
@@ -7,131 +7,80 @@
-  private static final int KEYCLOAK_PORT = 9784;
-  private static final String KEYCLOAK_RELATIVE_PATH = "/auth";
+  private static final int KEYCLOAK_PORT = 8080;
   private static final String KEYCLOAK_USER = "admin";
   private static final String KEYCLOAK_PASSWORD = "admin";
   private static final Network NETWORK = Network.newNetwork();
 
-  static GenericContainer<?> getPostgres() {
-    return new GenericContainer<>(DockerImageName.parse("postgres:16.6-alpine"))
-        .withNetwork(NETWORK)
-        .withNetworkAliases(POSTGRES_HOST)
-        .withExposedPorts(POSTGRES_PORT)
-        .withEnv("POSTGRES_DB", IDENTITY_DATABASE_NAME)
-        .withEnv("POSTGRES_USER", IDENTITY_DATABASE_USERNAME)
-        .withEnv("POSTGRES_PASSWORD", IDENTITY_DATABASE_PASSWORD)
-        .withLogConsumer(new Slf4jLogConsumer(LoggerFactory.getLogger("postgres:16.6-alpine")))
-        .withStartupTimeout(Duration.ofMinutes(5));
-  }
-
-  static GenericContainer<?> getKeycloak(final GenericContainer<?> postgres) {
-    return new GenericContainer<>("bitnami/keycloak:25.0.2")
+  static KeycloakContainer getKeycloak() {
+    return new KeycloakContainer()
+        .withEnv("KEYCLOAK_ADMIN", KEYCLOAK_USER)
+        .withEnv("KEYCLOAK_ADMIN_PASSWORD", KEYCLOAK_PASSWORD)
+        .withEnv("KEYCLOAK_DATABASE_VENDOR", "dev-mem")
         .withNetwork(NETWORK)
         .withNetworkAliases(KEYCLOAK_HOST)
-        .withExposedPorts(KEYCLOAK_PORT)
-        .dependsOn(postgres)
-        .withEnv("KEYCLOAK_HTTP_PORT", Integer.toString(KEYCLOAK_PORT))
-        .withEnv("KEYCLOAK_HTTP_RELATIVE_PATH", KEYCLOAK_RELATIVE_PATH)
-        .withEnv("KEYCLOAK_ADMIN_USER", KEYCLOAK_USER)
-        .withEnv("KEYCLOAK_ADMIN_PASSWORD", KEYCLOAK_PASSWORD)
-        .withEnv("KEYCLOAK_DATABASE_HOST", POSTGRES_HOST)
-        .withEnv("KEYCLOAK_DATABASE_NAME", IDENTITY_DATABASE_NAME)
-        .withEnv("KEYCLOAK_DATABASE_USER", IDENTITY_DATABASE_USERNAME)
-        .withEnv("KEYCLOAK_DATABASE_PASSWORD", IDENTITY_DATABASE_PASSWORD)
-        .withLogConsumer(new Slf4jLogConsumer(LoggerFactory.getLogger("bitnami/keycloak:25.0.2")))
+        .withExposedPorts(KEYCLOAK_PORT, 9000)
         .waitingFor(
             new HttpWaitStrategy()
-                .forPort(KEYCLOAK_PORT)
-                .forPath(KEYCLOAK_RELATIVE_PATH)
-                .allowInsecure()
+                .forPort(8080)
+                .forPath("/")
                 .forStatusCode(200)
-                .withReadTimeout(Duration.ofSeconds(10))
-                .withStartupTimeout(Duration.ofMinutes(5)));
+                .withStartupTimeout(Duration.ofMinutes(2)));
   }
 
-  static GenericContainer<?> getManagementIdentity(
-      final GenericContainer<?> postgres, final GenericContainer<?> keycloak) {
+  static GenericContainer<?> getManagementIdentity(final KeycloakContainer keycloak) {
     return new GenericContainer<>(DockerImageName.parse("camunda/identity:SNAPSHOT"))
```

Comment:
💭  I wonder whether we want to bind this to a release version for the upcoming 8.8 stable branch, but we can handle that in a follow-up given this is already the status quo anyway

Reply:
Yeah, good point. I think we can follow-up on this